tests/resources/Vsphere-Util.robot

```diff
@@ -190,9 +190,10 @@ Check Delete Success
     Log  ${out}
     Should Not Contain  ${out}  ${name}
 
-Gather Logs From ESX Server
-    Environment Variable Should Be Set  TEST_URL
+Gather vSphere Logs
+    Log To Console  Collecting vSphere logs...
     ${out}=  Run  govc logs.download
```

Comment:
This may require a list of hosts as well for general usage, although in this case the host in question is offline so I'm fine with just a vc-support bundle. Let me ask in the issue what hosts we should be collecting for given the host in question is offline.

Reply:
TJ Chen confirmed that we only need the vc-support bundle.